bitnami/opensearch/templates/_helpers.tpl

```diff
@@ -0,0 +1,679 @@
+{{/*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{/* vim: set filetype=mustache: */}}
+
+{{/*
+Return the proper OS image name
+*/}}
+{{- define "opensearch.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "opensearch.imagePullSecrets" -}}
+{{ include "common.images.pullSecrets" (dict "images" (list .Values.image .Values.sysctlImage .Values.volumePermissions.image) "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper sysctl image name
+*/}}
+{{- define "opensearch.sysctl.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.sysctlImage "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper image name (for the init container volume-permissions image)
+*/}}
+{{- define "opensearch.volumePermissions.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.volumePermissions.image "global" .Values.global) }}
+{{- end -}}
+
+
+{{/*
+Name for the Opensearch service
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+Required for the Kibana subchart to find Opensearch service.
+*/}}
+{{- define "opensearch.service.name" -}}
+{{- if .Values.global.kibanaEnabled -}}
+    {{- $name := .Values.global.opensearch.service.name -}}
```

Comment:
No reference to it in values.yaml

Reply:
[8144ede](https://github.com/bitnami/charts/pull/17509/commits/8144edebd19a8e8329cf21d424c37119b13120d8)